akka/shopping-cart-service-scala/build.sbt

```diff
@@ -0,0 +1,109 @@
+name := "shopping-cart-service"
+version := "1.0"
+
+organization := "com.lightbend.akka.samples"
+organizationHomepage := Some(url("https://akka.io"))
+licenses := Seq(("CC0", url("https://creativecommons.org/publicdomain/zero/1.0")))
+
+scalaVersion := "2.12.12"
```

Comment:
Downgrading to Scala 2.12 since Gatling does not support 2.13 yet.

Reply:
this will probably cause trouble when we use Scala 2.13 specific things in upstream example

Reply:
I think we can live with that for a while. According to Gatling maintainers, they will release a Scala 2.13 compatible version in the next couple of months.